mackup/applications/transmission.cfg

```diff
@@ -4,3 +4,8 @@ name = Transmission
 [configuration_files]
 Library/Preferences/org.m0k.transmission.plist
 Library/Application Support/Transmission/blocklists
+.config/transmission/blocklists
+.config/transmission/resume
+.config/transmission/torrents
```

Comment:
Can you `ls -l` those folders? You don't want to sync binary files, e.g. torrents.